conda/plugins/manager.py

```diff
@@ -423,6 +423,7 @@ def invoke_health_checks(self, prefix: str, verbose: bool) -> None:
                 hook.action(prefix, verbose)
             except Exception as err:
                 log.warning(f"Error running health check: {hook.name} ({err})")
+                raise
                 continue
```

Comment:
This is superfluous if you are raising the exception. However, this will prevent the other checks from happening. You might want to accumulate `err` in a list, and _then_ raise with `CondaMultiError`.

Reply:
oh this was only for testing purposes. I forgot to remove it, sorry!